diff --git a/debian-devel/compat b/debian-devel/compat
index 48082f72f..b4de39476 100644
--- a/debian-devel/compat
+++ b/debian-devel/compat
@@ -1 +1 @@
-12
+11
diff --git a/debian-devel/control b/debian-devel/control
index 1abbda247..41addd80b 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -5,7 +5,7 @@ Maintainer: Sébastien Helleu <flashcode@flashtux.org>
 Build-Depends:
     asciidoctor (>= 1.5.4),
     ruby-pygments.rb,
-    debhelper (>= 12),
+    debhelper (>= 11),
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -14,8 +14,8 @@ Build-Depends:
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-3.0-dev,
-    php-dev, libphp-embed, libargon2-dev, libsodium-dev,
+    guile-2.2-dev,
+    php-dev, libphp-embed, libargon2-0-dev, libsodium-dev,
     libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
diff --git a/debian-stable/compat b/debian-stable/compat
index 48082f72f..b4de39476 100644
--- a/debian-stable/compat
+++ b/debian-stable/compat
@@ -1 +1 @@
-12
+11
diff --git a/debian-stable/control b/debian-stable/control
index 428f11ce9..e88f167b8 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -5,7 +5,7 @@ Maintainer: Emmanuel Bouthenot <kolter@debian.org>
 Build-Depends:
     asciidoctor (>= 1.5.4),
     ruby-pygments.rb,
-    debhelper (>= 12),
+    debhelper (>= 11),
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -14,8 +14,8 @@ Build-Depends:
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-3.0-dev,
-    php-dev, libphp-embed, libargon2-dev, libsodium-dev,
+    guile-2.2-dev,
+    php-dev, libphp-embed, libargon2-0-dev, libsodium-dev,
     libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
