diff --git a/debian-devel/compat b/debian-devel/compat
index 48082f72f..b4de39476 100644
--- a/debian-devel/compat
+++ b/debian-devel/compat
@@ -1 +1 @@
-12
+11
diff --git a/debian-devel/control b/debian-devel/control
index a5d24b6a8..81ae98045 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -5,24 +5,23 @@ Maintainer: Sébastien Helleu <flashcode@flashtux.org>
 Build-Depends:
     asciidoctor (>= 1.5.4),
     ruby-pygments.rb,
-    debhelper (>= 12),
+    debhelper (>= 11),
     cmake, pkg-config,
-    libncurses-dev,
+    libncursesw5-dev,
     gem2deb,
     libperl-dev,
     python3-dev,
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-3.0-dev,
-    php-dev, libphp-embed, libargon2-dev, libsodium-dev,
+    guile-2.2-dev,
+    php-dev, libphp-embed, libargon2-0-dev, libsodium-dev,
     libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
     libgnutls28-dev,
     libzstd-dev,
-    zlib1g-dev,
-    libcjson-dev
+    zlib1g-dev
 Standards-Version: 4.6.2
 Homepage: https://weechat.org/
 Vcs-Git: https://salsa.debian.org/kolter/weechat.git
diff --git a/debian-devel/rules b/debian-devel/rules
index cab713c93..d2756333b 100755
--- a/debian-devel/rules
+++ b/debian-devel/rules
@@ -8,6 +8,7 @@ override_dh_auto_configure:
 	dh_auto_configure --buildsystem=cmake -- \
 		-DCMAKE_INSTALL_PREFIX:FILEPATH=/usr \
 		-DLIBDIR=/usr/lib/${DEB_HOST_MULTIARCH} \
+		-DENABLE_CJSON:BOOL=OFF \
 		-DENABLE_DOC:BOOL=ON \
 		-DENABLE_MAN:BOOL=ON \
 		-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo \
diff --git a/debian-stable/compat b/debian-stable/compat
index 48082f72f..b4de39476 100644
--- a/debian-stable/compat
+++ b/debian-stable/compat
@@ -1 +1 @@
-12
+11
diff --git a/debian-stable/control b/debian-stable/control
index a75e6fee5..b39bf88dc 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -5,24 +5,23 @@ Maintainer: Emmanuel Bouthenot <kolter@debian.org>
 Build-Depends:
     asciidoctor (>= 1.5.4),
     ruby-pygments.rb,
-    debhelper (>= 12),
+    debhelper (>= 11),
     cmake, pkg-config,
-    libncurses-dev,
+    libncursesw5-dev,
     gem2deb,
     libperl-dev,
     python3-dev,
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-3.0-dev,
-    php-dev, libphp-embed, libargon2-dev, libsodium-dev,
+    guile-2.2-dev,
+    php-dev, libphp-embed, libargon2-0-dev, libsodium-dev,
     libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
     libgnutls28-dev,
     libzstd-dev,
-    zlib1g-dev,
-    libcjson-dev
+    zlib1g-dev
 Standards-Version: 4.6.2
 Homepage: https://weechat.org/
 Vcs-Git: https://salsa.debian.org/kolter/weechat.git
diff --git a/debian-stable/rules b/debian-stable/rules
index cab713c93..d2756333b 100755
--- a/debian-stable/rules
+++ b/debian-stable/rules
@@ -8,6 +8,7 @@ override_dh_auto_configure:
 	dh_auto_configure --buildsystem=cmake -- \
 		-DCMAKE_INSTALL_PREFIX:FILEPATH=/usr \
 		-DLIBDIR=/usr/lib/${DEB_HOST_MULTIARCH} \
+		-DENABLE_CJSON:BOOL=OFF \
 		-DENABLE_DOC:BOOL=ON \
 		-DENABLE_MAN:BOOL=ON \
 		-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo \
