diff --git a/debian-devel/control b/debian-devel/control
index 809e059..8fbf0ed 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -3,7 +3,6 @@ Section: net
 Priority: optional
 Maintainer: Sébastien Helleu <flashcode@flashtux.org>
 Build-Depends:
-    asciidoctor (>= 1.5.4),
     debhelper (>= 9),
     dh-exec,
     cmake, pkg-config,
diff --git a/debian-stable/control b/debian-stable/control
index 74d134c..e0e9c25 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -3,7 +3,6 @@ Section: net
 Priority: optional
 Maintainer: Emmanuel Bouthenot <kolter@debian.org>
 Build-Depends:
-    asciidoctor (>= 1.5.4),
     debhelper (>= 9),
     dh-exec,
     cmake, pkg-config,
