diff --git a/debian-devel/compat b/debian-devel/compat
index 48082f72f..f599e28b8 100644
--- a/debian-devel/compat
+++ b/debian-devel/compat
@@ -1 +1 @@
-12
+10
diff --git a/debian-devel/control b/debian-devel/control
index bf94a36f1..ae29ae9ec 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Sébastien Helleu <flashcode@flashtux.org>
 Build-Depends:
     asciidoctor (>= 1.5.4),
-    debhelper (>= 12),
+    debhelper (>= 10),
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -13,8 +13,8 @@ Build-Depends:
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-2.2-dev,
-    php-dev, libphp-embed, libargon2-dev, libsodium-dev,
+    guile-2.0-dev,
+    php-dev, libphp-embed,
     libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
diff --git a/debian-stable/compat b/debian-stable/compat
index 48082f72f..f599e28b8 100644
--- a/debian-stable/compat
+++ b/debian-stable/compat
@@ -1 +1 @@
-12
+10
diff --git a/debian-stable/control b/debian-stable/control
index ccf81cac8..3e6563e42 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Emmanuel Bouthenot <kolter@debian.org>
 Build-Depends:
     asciidoctor (>= 1.5.4),
-    debhelper (>= 12),
+    debhelper (>= 10),
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -13,8 +13,8 @@ Build-Depends:
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-2.2-dev,
-    php-dev, libphp-embed, libargon2-dev, libsodium-dev,
+    guile-2.0-dev,
+    php-dev, libphp-embed,
     libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
