diff --git a/debian-devel/compat b/debian-devel/compat
index 48082f72f..ec635144f 100644
--- a/debian-devel/compat
+++ b/debian-devel/compat
@@ -1 +1 @@
-12
+9
diff --git a/debian-devel/control b/debian-devel/control
index ffca1a17f..8ccf774c9 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -3,20 +3,17 @@ Section: net
 Priority: optional
 Maintainer: Sébastien Helleu <flashcode@flashtux.org>
 Build-Depends:
-    asciidoctor (>= 1.5.4),
-    debhelper (>= 12),
+    debhelper (>= 9),
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
     libperl-dev,
     python3-dev,
     libaspell-dev,
-    liblua5.3-dev,
+    liblua5.1-0-dev,
     tcl8.6-dev,
     guile-2.0-dev,
     libv8-dev [amd64 i386 armel armhf mips mipsel hurd-i386 kfreebsd-amd64 kfreebsd-i386],
-    php-dev, libphp-embed, libargon2-0-dev, libsodium-dev,
-    libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
     libgnutls28-dev,
@@ -47,7 +44,7 @@ Description: Fast, light and extensible chat client (metapackage)
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - script manager
   - spell checking
   - highly customizable and extensible
@@ -77,7 +74,7 @@ Description: Fast, light and extensible chat client - console client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - script manager
   - spell checking
   - highly customizable and extensible
@@ -108,7 +105,7 @@ Description: Fast, light and extensible chat client - headless client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - script manager
   - spell checking
   - highly customizable and extensible
@@ -246,20 +243,6 @@ Description: Fast, light and extensible chat client - JavaScript plugin
  .
  This package provides the JavaScript scripting API plugin.
 
-Package: weechat-devel-php
-Architecture: any
-Depends:
-    ${misc:Depends},
-    ${shlibs:Depends},
-    weechat-devel-curses (= ${binary:Version}) | weechat-devel-headless (= ${binary:Version}),
-    libphp-embed
-Conflicts: weechat-php
-Description: Fast, light and extensible chat client - PHP plugin
- WeeChat (Wee Enhanced Environment for Chat) is a fast and light chat client
- for many operating systems. Everything can be done with a keyboard.
- .
- This package provides the PHP scripting API plugin.
-
 Package: weechat-devel-doc
 Section: doc
 Architecture: all
diff --git a/debian-devel/weechat-devel-php.dirs b/debian-devel/weechat-devel-php.dirs
deleted file mode 120000
index e4853a588..000000000
--- a/debian-devel/weechat-devel-php.dirs
+++ /dev/null
@@ -1 +0,0 @@
-../debian-stable/weechat-php.dirs
\ No newline at end of file
diff --git a/debian-devel/weechat-devel-php.install b/debian-devel/weechat-devel-php.install
deleted file mode 120000
index e358b515a..000000000
--- a/debian-devel/weechat-devel-php.install
+++ /dev/null
@@ -1 +0,0 @@
-../debian-stable/weechat-php.install
\ No newline at end of file
diff --git a/debian-stable/compat b/debian-stable/compat
index 48082f72f..ec635144f 100644
--- a/debian-stable/compat
+++ b/debian-stable/compat
@@ -1 +1 @@
-12
+9
diff --git a/debian-stable/control b/debian-stable/control
index 8fd0bf1a6..9cbe9d127 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -3,20 +3,17 @@ Section: net
 Priority: optional
 Maintainer: Emmanuel Bouthenot <kolter@debian.org>
 Build-Depends:
-    asciidoctor (>= 1.5.4),
-    debhelper (>= 12),
+    debhelper (>= 9),
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
     libperl-dev,
     python3-dev,
     libaspell-dev,
-    liblua5.3-dev,
+    liblua5.1-0-dev,
     tcl8.6-dev,
     guile-2.0-dev,
     libv8-dev [amd64 i386 armel armhf mips mipsel hurd-i386 kfreebsd-amd64 kfreebsd-i386],
-    php-dev, libphp-embed, libargon2-0-dev, libsodium-dev,
-    libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
     libgnutls28-dev,
@@ -46,7 +43,7 @@ Description: Fast, light and extensible chat client (metapackage)
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - script manager
   - spell checking
   - highly customizable and extensible
@@ -75,7 +72,7 @@ Description: Fast, light and extensible chat client - console client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - script manager
   - spell checking
   - highly customizable and extensible
@@ -105,7 +102,7 @@ Description: Fast, light and extensible chat client - headless client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - script manager
   - spell checking
   - highly customizable and extensible
@@ -234,19 +231,6 @@ Description: Fast, light and extensible chat client - JavaScript plugin
  .
  This package provides the JavaScript scripting API plugin.
 
-Package: weechat-php
-Architecture: any
-Depends:
-    ${misc:Depends},
-    ${shlibs:Depends},
-    weechat-curses (= ${binary:Version}) | weechat-headless (= ${binary:Version}),
-    libphp-embed
-Description: Fast, light and extensible chat client - PHP plugin
- WeeChat (Wee Enhanced Environment for Chat) is a fast and light chat client
- for many operating systems. Everything can be done with a keyboard.
- .
- This package provides the PHP scripting API plugin.
-
 Package: weechat-doc
 Section: doc
 Architecture: all
diff --git a/debian-stable/weechat-php.dirs b/debian-stable/weechat-php.dirs
deleted file mode 100644
index 68457717b..000000000
--- a/debian-stable/weechat-php.dirs
+++ /dev/null
@@ -1 +0,0 @@
-usr/lib
diff --git a/debian-stable/weechat-php.install b/debian-stable/weechat-php.install
deleted file mode 100644
index 0a1b73563..000000000
--- a/debian-stable/weechat-php.install
+++ /dev/null
@@ -1 +0,0 @@
-usr/lib/*/weechat/plugins/php.so
