diff --git a/debian-devel/control b/debian-devel/control
index 50f1217..24daebc 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -9,7 +9,6 @@ Build-Depends:
     docbook-xsl,
     docbook-xml,
     debhelper (>= 9),
-    dh-exec,
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -17,12 +16,12 @@ Build-Depends:
     python-dev,
     libaspell-dev,
     liblua5.1-0-dev,
-    tcl8.6-dev,
+    tcl8.5-dev,
     guile-2.0-dev,
-    libv8-dev [amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel],
+    libv8-dev,
     libcurl4-gnutls-dev,
-    libgcrypt20-dev,
-    libgnutls28-dev,
+    libgcrypt11-dev,
+    libgnutls-dev,
     zlib1g-dev
 Standards-Version: 3.9.6
 Homepage: https://weechat.org/
diff --git a/debian-stable/control b/debian-stable/control
index f9e9aa0..919202a 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -9,7 +9,6 @@ Build-Depends:
     docbook-xsl,
     docbook-xml,
     debhelper (>= 9),
-    dh-exec,
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -17,12 +16,12 @@ Build-Depends:
     python-dev,
     libaspell-dev,
     liblua5.1-0-dev,
-    tcl8.6-dev,
+    tcl8.5-dev,
     guile-2.0-dev,
-    libv8-dev [amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel],
+    libv8-dev,
     libcurl4-gnutls-dev,
-    libgcrypt20-dev,
-    libgnutls28-dev,
+    libgcrypt11-dev,
+    libgnutls-dev,
     zlib1g-dev
 Standards-Version: 3.9.6
 Homepage: https://weechat.org/
diff --git a/debian-stable/weechat-plugins.install b/debian-stable/weechat-plugins.install
old mode 100755
new mode 100644
index 198360f..980192f
--- a/debian-stable/weechat-plugins.install
+++ b/debian-stable/weechat-plugins.install
@@ -1,10 +1,8 @@
-#!/usr/bin/dh-exec
-
 usr/lib/weechat/plugins/aspell.so
 usr/lib/weechat/plugins/exec.so
 usr/lib/weechat/plugins/fifo.so
 usr/lib/weechat/plugins/guile.so
-[amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel] usr/lib/weechat/plugins/javascript.so
+usr/lib/weechat/plugins/javascript.so
 usr/lib/weechat/plugins/lua.so
 usr/lib/weechat/plugins/perl.so
 usr/lib/weechat/plugins/python.so
