diff --git a/debian-devel/control b/debian-devel/control
index 1abbda247..0db73f390 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -14,7 +14,7 @@ Build-Depends:
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-3.0-dev,
+    guile-2.2-dev,
     php-dev, libphp-embed, libargon2-dev, libsodium-dev,
     libxml2-dev,
     libcurl4-gnutls-dev,
diff --git a/debian-stable/control b/debian-stable/control
index 428f11ce9..dcf71da3e 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -14,7 +14,7 @@ Build-Depends:
     libaspell-dev,
     liblua5.3-dev,
     tcl8.6-dev,
-    guile-3.0-dev,
+    guile-2.2-dev,
     php-dev, libphp-embed, libargon2-dev, libsodium-dev,
     libxml2-dev,
     libcurl4-gnutls-dev,
