diff --git a/debian-devel/control b/debian-devel/control
index f8f027d8e..d382c2c9d 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -3,9 +3,7 @@ Section: net
 Priority: optional
 Maintainer: Sébastien Helleu <flashcode@flashtux.org>
 Build-Depends:
-    asciidoctor (>= 1.5.4),
     debhelper (>= 9),
-    dh-exec,
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -15,11 +13,10 @@ Build-Depends:
     liblua5.1-0-dev,
     tcl8.6-dev,
     guile-2.0-dev,
-    libv8-dev [amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel],
-    php7.1-dev, libphp7.1-embed, libxml2-dev,
+    libv8-dev,
     libcurl4-gnutls-dev,
-    libgcrypt20-dev,
-    libgnutls28-dev,
+    libgcrypt11-dev,
+    libgnutls-dev,
     zlib1g-dev
 Standards-Version: 3.9.8
 Homepage: https://weechat.org/
@@ -45,7 +42,7 @@ Description: Fast, light and extensible chat client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - scripts manager
   - spell checking
   - highly customizable and extensible
@@ -71,7 +68,7 @@ Description: Fast, light and extensible chat client - console client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - scripts manager
   - spell checking
   - highly customizable and extensible
@@ -178,17 +175,6 @@ Description: Fast, light and extensible chat client - javascript plugin
  .
  This package provides the JavaScript scripting API plugin.
 
-Package: weechat-devel-php
-Architecture: any
-Depends: ${misc:Depends}, ${shlibs:Depends}, weechat-devel-curses (= ${binary:Version}),
- libphp7.1-embed
-Conflicts: weechat-php
-Description: Fast, light and extensible chat client - php plugin
- WeeChat (Wee Enhanced Environment for Chat) is a fast and light chat client
- for many operating systems. Everything can be done with a keyboard.
- .
- This package provides the PHP scripting API plugin.
-
 Package: weechat-devel-doc
 Section: doc
 Architecture: all
diff --git a/debian-devel/weechat-devel-php.dirs b/debian-devel/weechat-devel-php.dirs
deleted file mode 120000
index e4853a588..000000000
--- a/debian-devel/weechat-devel-php.dirs
+++ /dev/null
@@ -1 +0,0 @@
-../debian-stable/weechat-php.dirs
\ No newline at end of file
diff --git a/debian-devel/weechat-devel-php.install b/debian-devel/weechat-devel-php.install
deleted file mode 120000
index e358b515a..000000000
--- a/debian-devel/weechat-devel-php.install
+++ /dev/null
@@ -1 +0,0 @@
-../debian-stable/weechat-php.install
\ No newline at end of file
diff --git a/debian-stable/control b/debian-stable/control
index a68d181e8..6c5ef314a 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -3,9 +3,7 @@ Section: net
 Priority: optional
 Maintainer: Emmanuel Bouthenot <kolter@debian.org>
 Build-Depends:
-    asciidoctor (>= 1.5.4),
     debhelper (>= 9),
-    dh-exec,
     cmake, pkg-config,
     libncursesw5-dev,
     gem2deb,
@@ -15,11 +13,10 @@ Build-Depends:
     liblua5.1-0-dev,
     tcl8.6-dev,
     guile-2.0-dev,
-    libv8-dev [amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel],
-    php7.1-dev, libphp7.1-embed, libxml2-dev,
+    libv8-dev,
     libcurl4-gnutls-dev,
-    libgcrypt20-dev,
-    libgnutls28-dev,
+    libgcrypt11-dev,
+    libgnutls-dev,
     zlib1g-dev
 Standards-Version: 3.9.8
 Homepage: https://weechat.org/
@@ -44,7 +41,7 @@ Description: Fast, light and extensible chat client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - scripts manager
   - spell checking
   - highly customizable and extensible
@@ -71,7 +68,7 @@ Description: Fast, light and extensible chat client - console client
   - 256 colors support
   - incremental text search
   - dynamic filtering of buffer content
-  - Perl, Python, Ruby, Lua, Tcl, Scheme, JavaScript and PHP scripting
+  - Perl, Python, Ruby, Lua, Tcl, Scheme and JavaScript scripting
   - scripts manager
   - spell checking
   - highly customizable and extensible
@@ -171,16 +168,6 @@ Description: Fast, light and extensible chat client - javascript plugin
  .
  This package provides the JavaScript scripting API plugin.
 
-Package: weechat-php
-Architecture: any
-Depends: ${misc:Depends}, ${shlibs:Depends}, weechat-curses (= ${binary:Version}),
- libphp7.1-embed
-Description: Fast, light and extensible chat client - php plugin
- WeeChat (Wee Enhanced Environment for Chat) is a fast and light chat client
- for many operating systems. Everything can be done with a keyboard.
- .
- This package provides the PHP scripting API plugin.
-
 Package: weechat-doc
 Section: doc
 Architecture: all
diff --git a/debian-stable/weechat-javascript.install b/debian-stable/weechat-javascript.install
old mode 100755
new mode 100644
index 428bb933e..e6cabcbec
--- a/debian-stable/weechat-javascript.install
+++ b/debian-stable/weechat-javascript.install
@@ -1,3 +1 @@
-#!/usr/bin/dh-exec
-
-[amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel] usr/lib/weechat/plugins/javascript.so
+usr/lib/weechat/plugins/javascript.so
diff --git a/debian-stable/weechat-php.dirs b/debian-stable/weechat-php.dirs
deleted file mode 100644
index 68457717b..000000000
--- a/debian-stable/weechat-php.dirs
+++ /dev/null
@@ -1 +0,0 @@
-usr/lib
diff --git a/debian-stable/weechat-php.install b/debian-stable/weechat-php.install
deleted file mode 100644
index 28b19cc1d..000000000
--- a/debian-stable/weechat-php.install
+++ /dev/null
@@ -1 +0,0 @@
-usr/lib/weechat/plugins/php.so
