diff --git a/debian-devel/control b/debian-devel/control
index f8f027d8e..1161649fc 100644
--- a/debian-devel/control
+++ b/debian-devel/control
@@ -16,7 +16,7 @@ Build-Depends:
     tcl8.6-dev,
     guile-2.0-dev,
     libv8-dev [amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel],
-    php7.1-dev, libphp7.1-embed, libxml2-dev,
+    php7.0-dev, libphp7.0-embed, libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
     libgnutls28-dev,
@@ -181,7 +181,7 @@ Description: Fast, light and extensible chat client - javascript plugin
 Package: weechat-devel-php
 Architecture: any
 Depends: ${misc:Depends}, ${shlibs:Depends}, weechat-devel-curses (= ${binary:Version}),
- libphp7.1-embed
+ libphp7.0-embed
 Conflicts: weechat-php
 Description: Fast, light and extensible chat client - php plugin
  WeeChat (Wee Enhanced Environment for Chat) is a fast and light chat client
diff --git a/debian-stable/control b/debian-stable/control
index a68d181e8..656e2def3 100644
--- a/debian-stable/control
+++ b/debian-stable/control
@@ -16,7 +16,7 @@ Build-Depends:
     tcl8.6-dev,
     guile-2.0-dev,
     libv8-dev [amd64 armel armhf hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 mips mipsel],
-    php7.1-dev, libphp7.1-embed, libxml2-dev,
+    php7.0-dev, libphp7.0-embed, libxml2-dev,
     libcurl4-gnutls-dev,
     libgcrypt20-dev,
     libgnutls28-dev,
@@ -174,7 +174,7 @@ Description: Fast, light and extensible chat client - javascript plugin
 Package: weechat-php
 Architecture: any
 Depends: ${misc:Depends}, ${shlibs:Depends}, weechat-curses (= ${binary:Version}),
- libphp7.1-embed
+ libphp7.0-embed
 Description: Fast, light and extensible chat client - php plugin
  WeeChat (Wee Enhanced Environment for Chat) is a fast and light chat client
  for many operating systems. Everything can be done with a keyboard.
